v2/pages/userInstanceQuestionExam/userInstanceQuestionExam.js

```diff
@@ -90,6 +91,27 @@ router.get('/', function(req, res, next) {
                 callback(null);
             });
         },
+        // We can probably combine the previous function with the following one, and eliminate sql.get_submission
+        function(callback) {
+            res.locals.showAllSubmissions = false;
+            submissionFilePath = path.join(res.locals.course.path, "questions", res.locals.question.directory, "submission.html");
```

Comment:
The showAllSubmissions flag will now be true if a submission exists; for problems which lack submission.html,, this will result in a list of submission panel-headers with no bodies, describing the submissions in order but not displaying them. Is this the behavior we want?